templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Guys, i made the changes that we talk about. 
Kafka now have a question type:service to select zookeeper stack/service.
Almost i add zookeeper service.